sale_financial_risk/models/sale.py

```diff
@@ -77,7 +77,11 @@ class SaleOrderLine(models.Model):
     )
     def _compute_risk_amount(self):
         for line in self:
-            if line.state != "sale":
+            skip = (
+                line.state != "cancel"
```

Comment:
You are changing the meaning of this.

Reply:
I'm just adding a condition to skip the computation when it's not needed. Please explain your point of view